### PR TITLE
Add wss4j dependency and support info to related document (#340)

### PR DESCRIPTION
**Pull request by EmreKp**
Hello,

I updated documentation and submitted PR related with #340 . I also pasted the note for support of which interceptors, may be helpful for other developers.

Thanks..

### CODE REVIEW

File: docs/modules/ROOT/pages/ws-security.adoc [hunk 1]
```diff
@@ -3,6 +3,19 @@
 
 The CXF framework's https://cxf.apache.org/docs/ws-security.html[WS-Security] implementation is based on https://ws.apache.org/wss4j/user_guide.html[WSS4J]. The example below shows how to integrate with WSS4J using interceptors.
 
+To use WSS4J interceptors in your code, firstly you should add CXF's WSS4J dependency (Replace `version` with CXF version used by Quarkus).
```
Comment (shumonsharif) [inline docs/modules/ROOT/pages/ws-security.adoc:6]: - You are adding CXF's **WS-Security** dependency _(not WSS4J)_
- I don't believe we need to say _"(Replace `version` with CXF version used by Quarkus)"_ .... for example, if someone is using the CXF BOM, then they don't need a version specified here at all.  Would suggest we remove this statement, along with the version tag in the XML block.  A basic understanding of Maven can be expected of the extension users, so I think we can omit this level of detail.

**Review by shumonsharif — CHANGES_REQUESTED** [review]
Thanks @EmreKp for this PR.  Would you kindly make the suggested updates?

File: docs/modules/ROOT/pages/ws-security.adoc (hunk 1 above)
Comment (EmreKp) [reply docs/modules/ROOT/pages/ws-security.adoc]: Thanks for suggestions, I changed version statement like "_you should add latest version_"  to satisfy any curiosities,  if it needs better statement, please let me know.

**Comment by EmreKp** [comment]
Thanks for suggestions, I also changed version statement like "_you should add latest version_" to answer any questions, if it needs better statement, please let me know.

**Comment by shumonsharif** [comment]
> Thanks for suggestions, I also changed version statement like "_you should add latest version_" to answer any questions, if it needs better statement, please let me know.

Thanks @EmreKp Yes, I would suggest we remove that statement all together; adding the latest version is not accurate either.

**Comment by EmreKp** [comment]
> > Thanks for suggestions, I also changed version statement like "_you should add latest version_" to answer any questions, if it needs better statement, please let me know.
> 
> Thanks @EmreKp Yes, I would suggest we remove that statement all together; adding the latest version is not accurate either.

I removed it for now, maybe it can be added later if some users would have problem concerned with that.

**Pull request merged**